connect/pom.xml

```diff
@@ -14,8 +14,8 @@
   <packaging>pom</packaging>
 
   <properties>
-    <commons-codec.version>1.15</commons-codec.version>
     <connect.version.old>1.6.0</connect.version.old>
+    <version.httpclient>5.3.1</version.httpclient>
```

Comment:
🔧 Can we centralize this version property with the External Task Client's property to avoid updating the Apache HTTP Client at two places?